ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,66 @@
+from ConfigParser import SafeConfigParser
+import logging
+import time
+import unittest
+
+logger = logging.getLogger('example')
+logging.basicConfig(level=logging.DEBUG)
+REFRESH_TIME = 15 * 60                           # 15 minutes
+
+
+class Config(object):
+
+    def __new__(cls, *args):
+        if not hasattr(cls, '_instance'):        # if there is not instance
+            cls._instance = object.__new__(cls)  # create it
+        return cls._instance                     # else return it
+
+    def __init__(self, path, logger):
+        self.config = {}                         # dictionary, contains configs
+        self.update_time = time.time() + REFRESH_TIME  # time of living
+        self.path = path                         # path to file (temporary)
+        self.logger = logger
+        self.logger.debug('Initialized instance at: %s', time.time())
+
+    def get_config(self):                        # method which checks if we
+        self.logger.debug('Check if need to update at %s', time.time())
+        if self.update_time < time.time():       # need to update configs
+            self.config = {}                     # nullify configs dictionary
+            self.update_time = time.time() + REFRESH_TIME  # set time to update
+            self._parse_confs()                  # parse config file
+        return self.config
+
+    def _parse_confs(self):
+        self.logger.debug('Parsed ecomap.conf at %s', (time.time()))
+        config = SafeConfigParser()              # create config object
+        config.readfp(open(self.path))           # read file
+        sections = config.sections()             # get sections
+        for section in sections:                 # for each section
+            for (key, value) in config.items(section):   # for each key/value
+                if value and key != 'password':
```

Comment:
can we use here a regexp for different types of 'password' key?
for example:
import re
p = re.compile(ur'(p)(a)?s*(w)?(d?)(or)?d?$', re.IGNORECASE | re.UNICODE)
test_str = u"pass\n"
re.search(p, test_str)